tla-bmcmt/src/test/scala/at/forsyte/apalache/tla/bmcmt/TestTrivialTypeFinder.scala

```diff
@@ -786,7 +776,9 @@ class TestTrivialTypeFinder extends RewriterBase {
     assert(FinSetT(IntT()) == typeFinder.compute(ex))
   }
 
-  test( "inferAndSave from the wild" ){
+  // Since the introduction of BmcOper.assign, the old assignments need to be transformed
```

Comment:
Could you introduce the existential in the test, instead of ignoring the test? The test is useful for regression.

Reply:
Done.